frontend/device/pocketbook/device.lua

```diff
@@ -112,14 +119,11 @@ function PocketBook:init()
             [KEY_OK] = "Press",
         },
         handleMiscEv = function(this, ev)
-            if ev.code == EVT_BACKGROUND then
-                self.isInBackGround = true
-                return "Suspend"
+            if ev.code == EVT_HIDE or ev.code == EVT_BACKGROUND then
+                return "SaveState"
             elseif ev.code == EVT_FOREGROUND then
-                if self.isInBackGround then
-                    self.isInBackGround = false
-                    return "Resume"
-                end
+                require("ui/uimanager"):setDirty('all', 'partial')
+                return "Foreground"
```

Comment:
The foreground event handler doesn't exist. It is intended?

Reply:
Yes, the intent is to translate it to something "human readable", and none of the existing frontent->uimanager events really fit. I'm not fond of spoofing 'Resume' and such when it's not - it's been source of trouble before. Better just name it properly, and let ui care (it bubbles down to `UIManager:sendEvent()` if not handled explicitly in manager/plugin).

Reply:
Ok, that makes sense.

Probably the only bit we might want from `onResume` is to update the footer to show the real battery percent

Reply:
Now looking at the plugins, the situation is rather unfortunate that those all treat Resume/Suspend as equivalent of foreground/background (launch/stop timers to update stuff etc). Guess there's no avoiding spoofing the event, instead one has to be careful to relegate its function only to "notify plugins", and don't *don't* hang actual Resume/Suspend state modifiers as the event may not come from these.

Blah.